spk/sickchill/src/service-setup.sh

```diff
@@ -1,4 +1,4 @@
-PYTHON_DIR="/usr/local/python3"
+PYTHON_DIR="/usr/local/python38"
```

Comment:
```suggestion
PYTHON_DIR="/var/packages/python38/target"
```

Reply:
`/usr/local/` is an old path, and we recommend to switch to the new path.

Reply:
noted.

Reply:
Sorry I missed this (by resolving your message was hidden). Thanks for the information.